### PR TITLE
Replace chrome branding strings in management_strings.grdp

### PR DESCRIPTION
**Pull request by simonhong**
This replaces chrome string in brave://management webui page.

Issue: https://github.com/brave/brave-browser/issues/454
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by mkarolin** [comment]
Looks like CI build failed for Android, but unrelated to these changes.

**Comment by simonhong** [comment]
@mkarolin Oh, thanks for checking. fix for that build failure is pushed in corresponding b-c PR(https://github.com/brave/brave-core/pull/3692).

File: lib/chromiumRebaseL10n.js [hunk 1]
```diff
@@ -11,8 +11,12 @@ const resetChromeStringFiles = () => {
   // Revert to original before string replacement for android_chrome_strings.grd
   // because original grd is overwritten with replaced one in ./src/brave.
   const srcDir = config.projects['chrome'].dir
-  const androidChromeStringsPath = path.resolve(path.join(srcDir, 'chrome', 'android', 'java', 'strings', 'android_chrome_strings.grd'))
-  util.run('git', ['checkout', androidChromeStringsPath], { cwd: srcDir })
+  const targetFilesForReset = []
+  targetFilesForReset.push(path.resolve(path.join(srcDir, 'chrome', 'android', 'java', 'strings', 'android_chrome_strings.grd')))
+  targetFilesForReset.push(path.resolve(path.join(srcDir, 'components', 'management_strings.grdp')))
```
Comment (darkdh) [inline lib/chromiumRebaseL10n.js]: why do we need to do this?
I have a similar PR replacing `bookmark_bar_strings.grdp` which is also part of `components_strings.grd`
https://github.com/brave/brave-browser/pull/6344

Comment (darkdh) [reply lib/chromiumRebaseL10n.js]: cc @petemill

Comment (mkarolin) [reply lib/chromiumRebaseL10n.js]: Typically, it's a manual step in the rebase process: https://github.com/brave/brave-browser/wiki/Chromium-rebases#fix-strings

**Pull request merged**